src/plan/generational/global.rs

```diff
@@ -172,7 +174,7 @@ impl<VM: VMBinding> Gen<VM> {
             return self.nursery.trace_object::<T>(
                 trace,
                 object,
-                CopySemantics::PromoteMature,
+                Some(CopySemantics::PromoteMature),
```

Comment:
Should this be called `PromoteNursery` or `PromoteToMature`. This confused me for a sec why we need to promote a mature object.